shared/CMakeLists.txt

```diff
@@ -34,10 +34,16 @@ set(SHARED_SRC
 		lib_aoi.h
 		lib_battery.cpp
 		lib_battery.h
+        lib_battery_capacity.cpp
```

Comment:
Looks like there are some tabs/spaces issues here. Is it worth fixing the whole list so the indentation isn't wonky?

Reply:
Oh that's strange as it didn't show up on my computer. Fixed!